cmp/grid/Grid.ts

```diff
@@ -367,20 +351,78 @@ class GridLocalModel extends HoistModel {
         };
     }
 
+    //----------------------
+    // Row Height Management
+    //----------------------
+    @computed
```

Comment:
Am I correct that we are using `@computed` specifically for its caching benefit?

Reply:
yes, exactly